.env.development

```
@@ -7,6 +7,7 @@ LOGIN_URL=http://localhost:18000/login
 LOGOUT_URL=http://localhost:18000/login
 REFRESH_ACCESS_TOKEN_ENDPOINT=http://localhost:18000/login_refresh
 MARKETING_SITE_BASE_URL='http://localhost:5335/'
+AUTH_BASE_URL='http://localhost:1999'
```

I think this name should be specific giving that auth is being handled in onboarding component as well. so its kinda confusing.
```suggestion
AUTHN_MFE_URL='http://localhost:1999'
```